traffic_control/clients/python/pylint.rc

```diff
@@ -259,7 +259,7 @@ redefining-builtins-modules=six.moves,past.builtins,future.builtins,builtins,io
 
 # Format style used to check logging format string. `old` means using %
 # formatting, while `new` is for `{}` formatting.
-logging-format-style=new
+logging-format-style=old
```

Comment:
shouldn't we let it be new?

Reply:
The new logging format isn't actually supported, Python the project has said they're just going to continue using the old format to avoid breaking people. I don't remember specifics, but I do remember being extremely annoyed by an inability to use the "new" format.

Reply:
copy!